packages/sx05re/emuelec/config/emuelec/scripts/show_splash.sh

```diff
@@ -14,42 +14,53 @@ DEFAULTSPLASH="/storage/.config/splash/splash-1080.png"
 VIDEOSPLASH="/usr/config/splash/emuelec_intro_1080p.mp4"
 DURATION="5"
 
+# we make sure the platform is all lowercase
+PLATFORM=${PLATFORM,,}
+
 case $PLATFORM in
- "ARCADE"|"FBA"|"NEOGEO"|"MAME"|CPS*)
-   PLATFORM="ARCADE"
+ "arcade"|"fba"|"neogeo"|"mame"|cps*)
+   PLATFORM="arcade"
   ;;
- "RETROPIE")
+ "retropie")
    # fbterm does not like the splash screen 
    exit 0
   ;;
 esac
 
 if [ "$PLATFORM" == "intro" ] || [ "$PLATFORM" == "exit" ]; then
 	SPLASH=${DEFAULTSPLASH}
-elif [ "$PLATFORM" == "default" ]; then
-	SPLASH=${GAMELOADINGSPLASH}
+#elif [ "$PLATFORM" == "default" ]; then
```

Comment:
Can you remove all the commented lines, if they are not needed

Reply:
Sure :)